targets/zephyr/Makefile.zephyr

```diff
@@ -47,6 +47,11 @@ endif
 
 INTERM     = build/$(BOARD)/obj-$(BOARD)
 OUTPUT     = build/$(BOARD)
+DOTCONFIG  = $(OUTPUT)/zephyr/.config
+
+include $(DOTCONFIG)
+override ARCH = $(subst ",,$(CONFIG_ARCH))
```

Comment:
To be honest I doubt the override is actually needed. However the equivalent code in the zephyr Makefile uses override so I decided it was better to act in the same manner as zephyr normally would.